app/js/checkAnswers.js

```diff
@@ -1,4 +1,35 @@
-const questionAmount = 30   // amount of questions
+const questionAmount = 30// amount of questions
+
+document.querySelector('#finish').addEventListener('click', function(e) {
+    var unanswered = questionAnswered()
+    if (unanswered == false) {
+        showResults()
+    }
+    else {
+        let flagList = document.getElementById('flag-list')
+        let unansweredList = document.getElementById('unanswered-list')
+        document.getElementById('modal-title').textContent = 'You have ' + unanswered.length + ' unanswered questions.'
+        flagList.innerHTML = ''
+        unansweredList.innerHTML = ''
+        Object.values(flaggedQuestions).forEach(function(isQuestionFlagged, qId) {
+            qId++
+            if (isQuestionFlagged) {
+                flagList.innerHTML += "<li>" + qId +"</li>"
+            }
```

Comment:
qId is the current builtin index inside forEach prototype, instead of incrementing it you could output (qId + 1) as this is what you'd like to output
`flagList.innerHTML += "<li>" + (qId + 1) +"</li>"`

Reply:
Thanks! c34b0db